libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -853,27 +858,74 @@ float AC_Avoid::distance_to_lean_pct(float dist_m)
     return 1.0f - (dist_m / _dist_max);
 }
 
+bool AC_Avoid::get_fence_roll_pitch_pct(float &roll_positive, float &roll_negative, float &pitch_positive, float &pitch_negative)
+{    
+    AC_Fence *fence = AP::fence();
```

Comment:
maybe just a const reference is enough ? we don't seems to use it other than for getter